libs/extended/src/System/Logger/Extended.hs

```diff
@@ -23,3 +28,21 @@ mkLogger' = Log.new
     . Log.setOutput Log.StdOut
     . Log.setFormat Nothing
     $ Log.defSettings
+
+newtype LoggerT m a = LoggerT {runLoggerT :: ReaderT Log.Logger m a}
+    deriving newtype
+        (Functor
```

Comment:
Do we have any particular typeclasses that we tend to derive? I just did the required ones for this usecase.

Reply:
Makes sense.  Whatever else is needed can be added when it is.